packages/install-expo-modules/src/utils/expoVersionMappings.ts

```diff
@@ -9,6 +9,11 @@ export interface VersionInfo {
 
 export const ExpoVersionMappings: VersionInfo[] = [
   // Please keep sdk versions in sorted order (latest sdk first)
+  {
+    expoSdkVersion: '46.0.0',
+    iosDeploymentTarget: '12.4',
+    reactNativeVersionRange: '>= 0.69.0',
```

Comment:
i think sdk 46 may also support react-native 0.68 or lower versions. i'm not clear whether we should aggressively promote latest sdk version or keep to use sdk 45 where we are confident supported.